mantaray/backend.py

```diff
@@ -200,7 +205,16 @@ def _connect_loop(self) -> None:
                 continue
 
             if self.password is not None:
-                self.send("CAP REQ sasl")
+                self.cap_req.append("sasl")
+
+            self.cap_req.append("away-notify")
+
+            self.pending_cap_count = len(
```

Comment:
This should be set to 0 in `__init__`. It's where people expect attributes to be defined. And yes, I know, there's `_apply_config()`, but that method is just below `__init__` for a reason.